agent/config/agent_limits.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul/lib"
+)
+
+// CheckLimitsFromMaxConnsPerClient check that value provided might be OK
+// return an error if values are not compatible
+func CheckLimitsFromMaxConnsPerClient(maxConnsPerClient int) error {
```

Comment:
This can be unexported (`checkLimitsFromMaxConnsPerClient`) now that it is called from inside the same package.